pages/partials/navbarInstitution.ejs

```diff
@@ -0,0 +1,11 @@
+<li class="nav-item">
```

Comment:
This could also conceivably go in `ee/partials`.

Reply:
As mentioned on the other comment, I'll leave this where it is for now.

Reply:
Agreed.